zotbins/ZotbinsCore
# Create mock weight class

## Description

Created the Mock Weight class that inherits from the IWeight class. Only have two member functions so far, the constructor, and the getWeight() overridden function.

## Type of change


## Testing and Verification Instructions

1. Build using the default environment
2. Verify that the build compiles successfully and that only the debug environment is built
3. Uploaded to the ESP32 to ensure that uploading still works

## Checklist:
